app/src/main/res/values/colors.xml

```diff
@@ -7,4 +7,6 @@
     <color name="teal_700">#FF018786</color>
 
     <color name="color_dark_blue">#114B70</color>
+    <color name="color_dark_gray">#2B2B2B</color>
+    <color name="color_dark_pink">#B7225B</color>
```

Comment:
giờ k đặt tên màu theo https://chir.ag/projects/name-that-color/ nữa à e

Reply:
Giờ e mới biết sự tồn tại của cái này